### PR TITLE
List REOSC Ecosystem (REOSC)

### PR DESCRIPTION
**Pull request by REOSC**
- Official project URL: [https://www.reosc.io/main.html]
- Official block explorer URL: [https://explorer.reosc.io]

### CODE REVIEW

File: core/src/main/java/bisq/core/locale/CurrencyUtil.java [hunk 1]
```diff
@@ -137,6 +137,7 @@ public static void setBaseCurrencyCode(String baseCurrencyCode) {
         result.add(new CryptoCurrency("XMR", "Monero"));
         result.add(new CryptoCurrency("MT", "Mycelium Token", true));
         result.add(new CryptoCurrency("NMC", "Namecoin"));
+        result.add(new CryptoCurrency("REOSC", "REOSC Ecosystem"));
```
Comment (blabno) [inline core/src/main/java/bisq/core/locale/CurrencyUtil.java]: Do not touch this file.

File: core/src/test/java/bisq/core/user/PreferencesTest.java [hunk 2]
```diff
@@ -124,7 +124,7 @@ public void testGetUniqueListOfCryptoCurrencies() {
 
         preferences.readPersisted();
 
-        assertEquals(13, preferences.getCryptoCurrenciesAsObservable().size());
+        assertEquals(14, preferences.getCryptoCurrenciesAsObservable().size());
```
Comment (blabno) [inline core/src/test/java/bisq/core/user/PreferencesTest.java]: Do not touch this file.

**Review by blabno — CHANGES_REQUESTED** [review]
NACK per comments

And no PRs from master.

**Comment by REOSC** [comment]
before I posted a commit here, I got the exchange to work on the local network and perform all the transactions. Without a change in the files indicated by you as "do not touch", the exchange did not work.

**Comment by blabno** [comment]
You clearly don't know what you are doing. Instead of reverting changes in CurrencyUtil you have removed the whole file.

**Comment by ManfredKarrer** [comment]
Close because of incompetent developers. We don't want to waste our time on such...

**Comment by REOSC** [comment]
ok, don't "wast"

**Pull request closed without merging**